Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM python:3.10.14-slim-bullseye
+
+LABEL    software="spades" \ 
+    author="Sebastian Bassi <sebastian@toyoko.io>" \
+    base_image="python:3.10.14-slim-bullseye" \ 
+    container="spades" \ 
+    about.summary="genome assembler for single-cell and isolates data sets" \ 
+    about.home="https://ablab.github.io/spades/" \ 
+    software.version="4.0.0" \ 
+    upstream.version="4.0.0" \ 
+    version="1" \ 
+    extra.identifiers.biotools="spades" \ 
+    about.copyright="2011-2014 Saint-Petersburg Academic University" \ 
+    about.license="GPL-2+" \ 
+    about.license_file="/usr/share/doc/spades/copyright" \ 
+    extra.binaries="/usr/bin/metaspades,/usr/bin/metaspades.py,/usr/bin/plasmidspades,/usr/bin/plasmidspades.py,/usr/bin/rnaspades,/usr/bin/rnaspades.py,/usr/bin/spades,/usr/bin/spades.py,/usr/bin/truspades,/usr/bin/truspades.py" \ 
```

Comment:
I do not think this is for `4.0.0`:
 - There is no truspades anymore
 - There are more binaries there

Reply:
this was based in an older Dockerfile published by Biocontainers, will remove old lines